CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# [2.0.1](https://github.com/CESARBR/knot-fog-connector-knot-cloud/compare/v2.0.0...v2.0.1)
+
+### Bug Fixes
+
+- update dependency lodash to v4.17.13 ([#10](https://github.com/CESARBR/knot-fog-connector-knot-cloud/pull/10)) ([cdafff9](https://github.com/CESARBR/knot-fog-connector-knot-cloud/pull/10/commits/cdafff964152978a2f0b766322b8c79b71afd5f2))
```

Comment:
Let start with a capital letter like the others. Also, I think you can just put "Update dependencies to fix vulnerability issues" and the packages would go as sub-items, without specifying the commit they were updated. The reason is that if we start to do this manually, it'd be difficult to create the CHANGELOG for the next releases. We can solve this problem by using a tool to automate this process as I've investigated before. Do you agree?

Reply:
Yeah, totally! :smile: 
Here are some tools we could consider, actually :point_down: 

[standard-version](github.com/conventional-changelog/standard-version): standard-version, “automates versioning and CHANGELOG generation, with semver.org and conventionalcommits.org.

[semantic-release](semantic-release.gitbook.io/semantic-release): automates the whole package release workflow including: determining the next version number, generating the release notes and publishing the package.

Reply:
Yes, I know our CHANGELOG version could be improved according to these guidelines but I think we should keep as it is as a matter of time. However, about the sub-items point, you can go with the approach you suggested but starting with a capital letter.

Reply:
Oops! Fixed capitalization :smile: